guides/common/modules/proc_using-red-hat-insights.adoc

```diff
@@ -57,6 +57,20 @@ You can manually synchronize the recommendations using the following procedure:
 
 If you have not set up the API token, you are prompted to create one before using this page.
 
+.Removing hosts from Inventory of Red{nbsp}Hat Insights when they are removed from {Project}
+When hosts are removed from {Project}, they can also be removed from the inventory of Red{nbsp}Hat Insights, either automatically or manually.
+Use the following procedure to configure it automatically.
+
+.Prerequisite
+* You must have administrator privilege for the organization to enable this setting.
```

Comment:
Since the setting is Satellite - wide, you have to be Satellite administrator, not org admin.

Reply:
Added this one -https://github.com/theforeman/foreman-documentation/pull/2502#discussion_r1366857395